libraries/Microsoft.Bot.Builder.LanguageGeneration/Template.cs

```diff
@@ -31,8 +31,17 @@ public class Template
             this.Parameters = parameters ?? new List<string>();
             this.Body = templateBody ?? string.Empty;
             this.SourceRange = sourceRange;
+            Expressions = new List<ExpressionContext>();
         }
 
+        /// <summary>
+        /// Gets or sets expression context list.
+        /// </summary>
+        /// <value>
+        /// Expression context list.
+        /// </value>
+        public IList<ExpressionContext> Expressions { get; set; }
```

Comment:
I prefer consistent naming pattern, if you call this ExpressionContext expression, i think this indicate ExpressionContext is not a good name?